src/NotificationService/NotificationService.csproj

```diff
@@ -15,10 +15,10 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" />
     <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" />
     <PackageReference Include="Microsoft.Extensions.Configuration.CommandLine" />
-    <PackageReference Include="Microsoft.Extensions.Http" />
+    <PackageReference Include="Microsoft.Extensions.Http" VersionOverride="8.0.0" />
```

Comment:
Waarom hier version override i.p.v de versie in Directory.Packages.Props aanpassen?

Reply:
Dit was een van mijn docker errors dat hij bij deze 2 gewoon simpelweg NIET naar directory.packages.props wou kijken ik zal even kijken of het nu zonder override werkt.